### PR TITLE
refactor: Improve WMS layers table UI

### LINKED ISSUE #2008: Layer titles unreadable in the add-data layer list
..due to long names and un-scrollable content

image

### PR DESCRIPTION
**Pull request by FilipLeitner**
fixes #2008

### CODE REVIEW

**Comment by jmacura** [comment]
Nice improvement, as a simple fix it's great.
image

I suggest a bit more complex improvement: additionally to these changes, add (click) action to each line, which would expand the text in full length (all cells, single line). What do you think?

**Comment by raitisbe** [comment]
The name should be somewhere in case the titles for 2 layers are the same, but names differ. Name as hover tooltip maybe?

**Comment by FilipLeitner** [comment]
> The name should be somewhere in case the titles for 2 layers are the same, but names differ. Name as hover tooltip maybe?

Hmm I mean now 

> The name should be somewhere in case the titles for 2 layers are the same, but names differ. Name as hover tooltip maybe?

Added name to the tooltip. It feels a bit tricky but works for me on Chrome, Firefox and also Edge so I hope itll be alright.

**Review by raitisbe — APPROVED** [review]

File: projects/hslayers/src/components/add-data/url/wms/add-data-url-wms.service.ts [hunk 1]
```diff
@@ -100,6 +100,18 @@ export class HsAddDataUrlWmsService {
     this.getDimensionValues = hsDimensionService.getDimensionValues;
   }
 
+  expandTableRow(e) {
+    const action = e.target.className.includes('hs-wms-expandedRow')
```
Comment (raitisbe) [inline projects/hslayers/src/components/add-data/url/wms/add-data-url-wms.service.ts:104]: Was there no angularish way to do this?

**Review by raitisbe — APPROVED** [review]

File: projects/hslayers/src/components/add-data/url/wms/add-data-url-wms.service.ts (hunk 1 above)
Comment (FilipLeitner) [reply projects/hslayers/src/components/add-data/url/wms/add-data-url-wms.service.ts:104]: What exactly?

**Review by raitisbe — APPROVED** [review]

**Pull request merged**